spec/audience_spec.rb

```diff
@@ -193,4 +195,130 @@
     expect(customer_attr).to have_received(:exact_evaluator).with(audience_3468206645_condition1).once
     expect(customer_attr).to have_received(:exact_evaluator).with(audience_3468206645_condition2).once
   end
+
+  it 'should return nil when audience not found' do
+    experiment = config.experiment_key_map['test_experiment_with_audience']
+    user_attributes = {
+      'browser_type' => 5.5
+    }
+    experiment['audienceIds'] = %w[11110]
+
+    expect(Optimizely::Audience.user_in_experiment?(config,
+                                                    experiment,
+                                                    user_attributes)).to be false
+    expect(spy_logger).to have_received(:log).once.with(
+      Logger::DEBUG,
+      "Evaluating audiences for experiment 'test_experiment_with_audience': #{experiment['audienceIds']}."
```

Comment:
Can we avoid string interpolation in these tests? It'd be helpful to show what the final strings look like, especially since we're relying on Ruby's string representation of the underlying hash.